slides/quickstart/index.html

```diff
@@ -122,26 +87,21 @@
       function listSlides() {
         gapi.client.slides.presentations.get({
           presentationId: '1EAYk18WDjIG-zp_0vLm3CsfQh_i8eXc67Jo2O9C6Vuc'
-        }).then(function(response) {
-          var presentation = response.result;
-          var length = presentation.slides.length;
-          appendPre('The presentation contains ' + length + ' slides:');
-          for (i = 0; i < length; i++) {
-            var slide = presentation.slides[i];
-            appendPre('- Slide #' + (i + 1) + ' contains ' +
-                slide.pageElements.length + ' elements.')
-          }
-        }, function(response) {
-          appendPre('Error: ' + response.result.error.message);
+        }).then((response) => {
+          const presentation = response.result;
+          const length = presentation.slides.length;
+          appendPre(`The presentation contains ${length} slides:`);
+          presentation.slides.map((slide, i) => {
+            appendPre(`- Slide #${(i + 1)} contains ${slide.pageElements.length} elements.`);
+          });
+        }).catch((response) => {
```

Comment:
It looks like only a few of the quickstarts include the `catch`. Should we be including it for all of them?